.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+    "printWidth": 120,
+    "bracketSameLine": true,
+    "singleQuote": true,
+    "trailingComma": "none",
+    "arrowParens": "always",
+    "tabWidth": 4
```

Comment:
Will this enforce 4 spaces on the files when running prettier? The command files look like 2 spaces

Reply:
ugh my eslint was so confused from all my changes it wasn't overriding prettier 😓 fixed back to 4 spaces now. this is just setting tab width as 4 spaces here.  we can remove tho if we don't like that.